src/Microsoft.Graph/Models/Generated/ChatMessageMention.cs

```diff
@@ -19,8 +19,7 @@ namespace Microsoft.Graph
     /// The type ChatMessageMention.
     /// </summary>
     [JsonObject(MemberSerialization = MemberSerialization.OptIn)]
-    [JsonConverter(typeof(DerivedTypeConverter))]
-    public partial class ChatMessageMention
+    public partial class ChatMessageMention : Entity
```

Comment:
It looks like we are adding a base type to a complex type that shouldn't have one. This is happening in our XSLT transform.

According to the [live metadata](https://graph.microsoft.com/beta/$metadata), `chatMessageMention` is a complex type with no base type, but in the [clean metadata](https://raw.githubusercontent.com/microsoftgraph/msgraph-metadata/master/clean_v10_metadata/cleanMetadataWithDescriptionsv1.0.xml), we seem to be adding a base type of `entity` to it. This leads to a property conflict since both `entity` and `chatMessageMention` have an `id` property of different types, a `string` for `entity` and  `int32` for  `chatMessageMention`.

#### Live Metadata
![image](https://user-images.githubusercontent.com/7061532/83090615-05989380-a04e-11ea-9821-22ef18466e78.png)
#### Clean Metadata
![image](https://user-images.githubusercontent.com/7061532/83090674-2a8d0680-a04e-11ea-805b-6073f2f0c424.png)

The same applies to the clean v1.0 metadata.